Library/Formula/rmlint.rb

```diff
@@ -0,0 +1,28 @@
+class Rmlint < Formula
+  desc "remove duplicates and lint from your filesystem"
+  homepage "http://rmlint.rtfd.org"
+  url "https://github.com/sahib/rmlint/archive/v2.4.3.tar.gz"
+  sha256 "499c38449038c30b7704760d1251f0098fb28f6037e54c434ef24d6d18d0b5a5"
+
+  depends_on "glib" => :build
+  depends_on "scons" => :build
+  depends_on "gettext" => [:build, :recommended]
+  depends_on "sphinx" => [:build, :optional]
+  depends_on "gtk+" => :recommended # for gui
```

Comment:
Presumably it wouldn't make sense to opt-out of `gtk+` but leave `pygobject` enabled?